tests/test-class-opengraph.php

```diff
@@ -330,11 +334,12 @@ public function test_image_IS_SINGULAR_AND_HAS_featured_image_AND_HAS_WRONG_size
 
 		$class_instance->opengraph();
 
-		$output = ob_get_contents();
+		$output = ob_get_clean();
```

Comment:
`ob_get_contents(); ob_end_clean();` is exactly same as just `ob_get_clean();`, I just changed to a more compact way to write same action. Not sure what you mean by "risky".